pki/ejbca/api-docs/apiary.apib

```diff
@@ -0,0 +1,490 @@
+FORMAT: 1A
+
+# EJBCA NODE API
```

Comment:
EJBCA NODE is not intuitive, I'd change for something like REST Wrapper for EBJCA or EJBCA REST